wafer/pages/models.py

```diff
@@ -8,13 +9,30 @@
 from django.urls import reverse
 from django.utils.translation import ugettext_lazy as _
 
-from markitup.fields import MarkupField, render_func
+from markitup.fields import MarkupField
 
 from wafer.menu import refresh_menu_cache
 
 logger = logging.getLogger(__name__)
 
 
+class PageMarkupField(MarkupField):
+    """MarkupField that uses our own render function"""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        dotted_path, kwargs = settings.WAFER_PAGE_MARKITUP_FILTER
+        module, func = dotted_path.rsplit('.', 1)
+        func = getattr(__import__(module, {}, {}, [func]), func)
+        self.render_func = partial(func, **kwargs)
```

Comment:
This feels like something that'd ideally be made configurable in django-markitup, itself. I'll pursue that next.